src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -14,6 +15,20 @@ export default class OrganizationDeveloperSettings extends AsyncView {
     return [['applications', `/organizations/${orgId}/sentry-apps/`]];
   }
 
+  removeApp = app => {
+    const api = this.api;
```

Comment:
I don't know if we really need to put this in another variable if it's just `this.api`.